src/olympia/addons/tests/test_search.py

```diff
@@ -27,3 +30,28 @@ def test_extract_attributes(self):
         extracted = self._extract()
         for attr in self.attrs:
             eq_(extracted[attr], getattr(self.addon, attr))
+
+    def test_extract_translations(self):
+        translations_name = {
+            'en-US': u'Name in ënglish',
+            'es': u'Name in Español',
+            'it': None,  # Empty name should be ignored in extract.
+        }
+        translations_description = {
+            'en-US': u'Description in ënglish',
+            'es': u'Description in Español',
+            'fr': '',  # Empty description should be ignored in extract.
+        }
+        self.addon = Addon.objects.get(pk=3615)
+        self.addon.name = translations_name
+        self.addon.description = translations_description
+        self.addon.save()
+        extracted = self._extract()
+        eq_(extracted['name_translations'], [
```

Comment:
I know the whole file is using `eq_` and stuff but can we use `assert` statements for new code please? We wanted to get rid of nose for quite some time